internal/controllers/builder.go

```diff
@@ -41,6 +60,7 @@ func (c *ControllerBuilder) OnChange(_ context.Context, _ store.RStore, _ store.
 func (c *ControllerBuilder) SetUp(ctx context.Context, st store.RStore) error {
 	mgr := c.tscm.GetManager()
 	client := c.tscm.GetClient()
+	_ = logger.Get(ctx)
```

Comment:
Is this being done for side-effects (panic if no logger attached)? If so, a comment might be warranted